### PR TITLE
[SEM-093]: FORM FIELDS FROM SEO SETTINGS

### PR DESCRIPTION
**Pull request by MadelenC**
We need to implement a component that contains the elements shown in the image, thus containing all the form fields, as well as integrating the editing component shown in the image.

### CODE REVIEW

**Comment by MadelenC** [comment]
https://ditmar.github.io/backoffice-openscience/branch-SEM-093/?path=/story/components-seosettingsform--default
this is for storybook

File: src/components/SEOSettingsForm/SEOSettingsForm.test.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { render, screen } from '@testing-library/react';
+import { describe, test, expect } from 'vitest';
+import SEOSettingsForm from './SEOSettingsForm';
+import '@testing-library/jest-dom';
```
Comment (Ditmar) [inline src/components/SEOSettingsForm/SEOSettingsForm.test.tsx:4]: por que necesitas importar jest, estamos usando vitest

File: src/components/SEOSettingsForm/SEOSettingsForm.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import styles from './SEOSettingsForm.module.scss';
+
+function SEOSettingsForm() {
+  return (
+    <form className={styles['seo-setting-form-one']}>
+      <div className={styles['seo-settings-form']}>
+        <div className={styles['form-group']}>
+          <label htmlFor="meta_title">
+            meta_title
+            <input id="meta_title" type="text" placeholder="Placeholder text" />
```
Comment (Ditmar) [inline src/components/SEOSettingsForm/SEOSettingsForm.tsx:10]: el input deberia ser un atomo y esperar los props para sus variantes

File: src/components/SEOSettingsForm/SEOSettingsForm.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        <div className={styles['form-group']}>
+          <label htmlFor="og_url">
+            og_url
+            <input id="og_url" type="text" placeholder="Placeholder text" />
+          </label>
+        </div>
+        <div className={styles['form-group']}>
+          <label htmlFor="og_image_height">
+            og_image_height
+            <input id="og_image_height" type="text" placeholder="Placeholder text" />
```
Comment (Ditmar) [inline src/components/SEOSettingsForm/SEOSettingsForm.tsx:28]: el input deberia ser un atomo y esperar los props para sus variantes

**Review by Ditmar — CHANGES_REQUESTED** [review]
lea los comentarios

**Comment by Ditmar** [comment]
igual no lo veo bien en el storybook en otras resoluciones 
image

**Comment by MadelenC** [comment]
https://ditmar.github.io/backoffice-openscience/branch-SEM-093/?path=/story/components-seosettingsform--default

**Comment by MadelenC** [comment]
https://ditmar.github.io/backoffice-openscience/branch-SEM-093/?path=/story/components-seosettingsform--default

File: src/components/SEOSettingsForm/SEOSettingsForm.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import type { StoryObj, Meta } from '@storybook/react';
+import SEOSettingsForm from './SEOSettingsForm';
+
+const meta: Meta<typeof SEOSettingsForm> = {
+  title: 'Components/SEOSettingsForm',
```
Comment (Ditmar) [inline src/components/SEOSettingsForm/SEOSettingsForm.stories.tsx]: this part of code should be on like a molecules

**Review by Ditmar — APPROVED** [review]

**Pull request merged**